pkg/controllers/dynakube/oneagent/oneagent_reconciler.go

```diff
@@ -167,6 +170,7 @@ func (r *Reconciler) createOneAgentTenantConnectionInfoConfigMap(ctx context.Con
 	err = query.CreateOrUpdate(*configMap)
 	if err != nil {
 		log.Info("could not create or update configMap for connection info", "name", configMap.Name)
+		conditions.SetKubeApiErrorCondition(r.dynakube.Conditions(), oaConditionType, err)
```

Comment:
```suggestion
		conditions.SetKubeApiErrorCondition(r.dynakube.Conditions(), oaConditionType, err)
```
that's what I mean: `conditions.SetKubeApiError` may be?